ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -143,6 +151,75 @@ export const UserRoleOptions = [
   { value: WorkspaceAccessLevel.OWNER, label: 'Owner' },
 ];
 
+interface ConflictProps {
+  reloadAction: () => void;
+  cancelAction: () => void;
+}
```

Comment:
By my reading, if these made it through by accident, things would silently fail (a user's click would do nothing). Perhaps we'd want something more like:
```suggestion
const invalidClickAction = () => alert('Crap, we screwed up. Please let support know you are seeing this message.');
interface ConflictProps {
  reloadAction: invalidClickAction;
  cancelAction: invalidClickAction;
```

Reply:
This is not an implementation, but the type signature: a function taking nothing and also returning nothing.